lib/rules/selector-pseudo-element-blacklist/index.js

```diff
@@ -0,0 +1,78 @@
+"use strict";
+
+const _ = require("lodash");
+const isStandardSyntaxRule = require("../../utils/isStandardSyntaxRule");
+const isStandardSyntaxSelector = require("../../utils/isStandardSyntaxSelector");
+const matchesStringOrRegExp = require("../../utils/matchesStringOrRegExp");
+const parseSelector = require("../../utils/parseSelector");
+const postcss = require("postcss");
+const report = require("../../utils/report");
+const ruleMessages = require("../../utils/ruleMessages");
+const validateOptions = require("../../utils/validateOptions");
+
+const ruleName = "selector-pseudo-element-blacklist";
+
+const messages = ruleMessages(ruleName, {
+  rejected: selector => `Unexpected pseudo-element "${selector}"`
+});
+
+const rule = function(blacklist) {
+  return (root, result) => {
+    const validOptions = validateOptions(result, ruleName, {
+      actual: blacklist,
+      possible: [_.isString]
```

Comment:
It should also check for `_.isRegExp` according to readme. It would be great to have RegExp in tests as well. Both as a string and as RegExp.

The same comment applies to the second rule.